person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+    @id = Random.rand(1..100)
+  end
+
+  def propper_age?
+    @age >= 18
+  end
```

Comment:
- [ ] Kindly make this method private as per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=Private%20method%20is_of_age%3F%20that%20returns%20true%20if%20%40age%20is%20greater%20or%20equal%20to%2018%20and%20false%20otherwise.).